src/styles/main.scss

```diff
@@ -1,3 +1,59 @@
-body {
+@import "./utils/variables";
+@import "./utils/mixins";
+
+.page {
   margin: 0;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  width: $calendarArea;
+  padding: 25px 10px 10px;
+  border: 1px solid #00acdc;
+
+  &__day {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    font-size: 30px;
+    @include cellSize($cell-size);
+    background-color: #eee;
+    border: 1px solid;
+    text-align: center;
+    box-sizing: border-box;
+    transition-duration: 0.25s;
+
+    &:hover {
+      cursor: pointer;
```

Comment:
Don't set cursor pointer in the :hover. Move it to main selector

Reply:
done